pynecone/config.py

```diff
@@ -198,6 +207,33 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+        # set overriden class attribute values as os env variables to avoid losing them
+        for key, value in dict(self).items():
+            key = key.upper() if key != "port" else "FRONTEND_PORT"
```

Comment:
Maybe we can rename the `--port` flag to `--frontend-port` to minimize confusion / having special cases. Would be a breaking change though